### PR TITLE
Address minor issues in Trilinos NOX wrappers

### PR DESCRIPTION
**Pull request by jppelteret**
Follow-up to #14332

### CODE REVIEW

**Review by peterrum — APPROVED** [review]
Thanks!

File: include/deal.II/trilinos/nox.h [hunk 1]
```diff
@@ -49,16 +49,16 @@ namespace TrilinosWrappers
    * // create nonlinear solver
    * TrilinosWrappers::NOXSolver<VectorType> solver(additional_data);
    *
-   * // set user functions to compute residual,
-   * // set up Jacobian, and to apply the inverse of
-   * // Jacobian; note that there are more functions that
-   * // can be set
+   * // Set user functions to compute residual,
+   * // set up the Jacobian, and to apply the inverse of
```
Comment (kronbichler) [inline include/deal.II/trilinos/nox.h:53]: Shouldn't we use
```suggestion
   * // to set up the Jacobian, and to apply the inverse of
```
for consistent use of prepositions within the sentence?

File: include/deal.II/trilinos/nox.h [hunk 2]
```diff
@@ -49,16 +49,16 @@ namespace TrilinosWrappers
...
+   * // Jacobian.
```
Comment (kronbichler) [inline include/deal.II/trilinos/nox.h:54]:
```suggestion
   * // the Jacobian.
```

File: include/deal.II/trilinos/nox.templates.h [hunk 3]
```diff
@@ -70,7 +70,7 @@ namespace TrilinosWrappers
         }
 
         /**
-         * Initialize every element of this vector with gamma.
+         * Initialize every element of this vector with @p gamma.
```
Comment (kronbichler) [inline include/deal.II/trilinos/nox.templates.h:73]: Given the change you made above, isn't `@p gamma.` also rendering `.` with the `tt` font? (Not that it matters much.)

Comment (jppelteret) [reply include/deal.II/trilinos/nox.templates.h:73]: Yes, I think that you're right. I've fixed this.

File: include/deal.II/trilinos/nox.h (hunk 1 above)
Comment (jppelteret) [reply include/deal.II/trilinos/nox.h:53]: Yes, you're correct. I've fixed both errors, and have adjusted the line breaks for this set of comments.

**Pull request merged**